MonoGame.Framework/Graphics/GraphicsDevice.cs

```diff
@@ -220,8 +220,13 @@ public GraphicsDevice(GraphicsAdapter adapter, GraphicsProfile graphicsProfile,
             _graphicsProfile = graphicsProfile;
             Setup();
             GraphicsCapabilities = new GraphicsCapabilities();
-            GraphicsCapabilities.Initialize(this);
-            Initialize();
+            GraphicsCapabilities.Initialize(this);
+
+#if WINDOWS
+            CorrectBackBufferSize();
+#endif
```

Comment:
We could do it with partial stuff, but I think it would be good if all platforms eventually implemented this method. The #if would be temporary. 

_Sent from my Motorola Moto G (5) using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_